src/Vegetation/BigLeaf/bigleaf_parameterizations.jl

```diff
@@ -0,0 +1,271 @@
+    APAR = PAR*(1-ρ_leaf)*(1-exp(-K*LAI*Ω))
+    return APAR
+end
+
+# 2. Photosynthesis, Farquhar model
+
+"""
+    rubisco_assimilation_C3(Vcmax::FT,
+                            ci::FT,
+                            cp::FT,
+                            Kc::FT,
+                            Ko::FT) where {FT}
+
+Computes the Rubisco limiting rate of photosynthesis for C3 plants (`a1`),
+as a function of the maximum rate of carboxylation of Rubisco (`Vcmax`), 
+the leaf internal carbon dioxide partial pressure (`ci`), 
+the CO2 compensation point (`cp`), and Michaelis-Menten parameters
+for CO2 and O2, respectively, (`Kc`) and (`Ko`).
+"""
+function rubisco_assimilation_C3(Vcmax::FT,
+                                 ci::FT,
+                                 cp::FT,
+                                 Kc::FT,
+                                 Ko::FT) where {FT}
+    a1 = Vcmax*(ci-cp)/(ci+Kc*(1+209/Ko))
+    return a1
+end
+
+"""
+    rubisco_assimilation_C4(Vcmax::FT) where {FT}
+
+Computes the Rubisco limiting rate of photosynthesis for C4 plants (`a1`),
+as equal to the maximum rate of carboxylation of Rubisco (`Vcmax`).
+"""
+function rubisco_assimilation_C4(Vcmax::FT) where {FT}
+    a1 = Vcmax
+    return a1
+end
+
+"""
+    light_assimilation_C3(J::FT,
+                          ci::FT,
+                          cp::FT) where {FT}
+
+Computes the electron transport limiting rate (`a2`) for C3 plants as a function of
+the rate of electron transport (`J`), the leaf internal carbon dioxide partial pressure (`ci`),
+and the CO2 compensation point (`cp`).
+
+"""
+function light_assimilation_C3(J::FT,
+                               ci::FT,
+                               cp::FT) where {FT}
+    a2 = J*(ci-cp)/4*(ci+2cp)
+    return a2
+end
+
+"""
+    light_assimilation_C4(J::FT) where {FT}
+
+Computes the electron transport limiting rate (`a2`) for C4 plants, as equal to
+the rate of electron transport (`J`).
+"""
+function light_assimilation_C4(J::FT) where {FT}
+    a2 = J
+    return a2
+end
+
+"""
+    net_photosynthesis(a1::FT,
+                       a2::FT,
+                       Rd::FT,
+                       β::FT) where {FT}
+
+Computes the total net carbon assimilation (`An`) as a function of 
+the Rubisco limiting factor (`a1`), the electron transport limiting rate (`a2`),
+dark respiration (`Rd`), and the moisture stress factor (`β`). 
+"""
+function net_photosynthesis(a1::FT,
+                            a2::FT,
+                            Rd::FT,
+                            β::FT) where {FT}
+    An = maximum([0, minimum([a1*β, a2]) - Rd])
+    return An
+end
+
+"""
+dark_respiration(Vcmax::FT,
+                 β::FT) where {FT}
+
+Computes dark respiration (`Rd`) as a function of the maximum rate of carboxylation of Rubisco (`Vcmax`),
+and the moisture stress factor (`β`). 
+"""
+function dark_respiration(Vcmax::FT,
+                          β::FT) where {FT}
+    Rd = 0.015*Vcmax*β
```

Comment:
same with the 0.015 factor

Reply:
same as above @braghiere 
Note I understand this might be purely empirical params, it's ok to just call it scaler or something